usr/src/cmd/svc/milestone/net-early-admin

```diff
@@ -189,7 +199,7 @@ if [[ -n "$dlmtu" ]]; then
         fatal "ERROR: Failed to set mtu on aggr $nic to $dlmtu"
 fi
 
-driver=${nic%%[0-9]*}
+driver=${nic%%+([0-9])}
```

Comment:
Why does this change?

Reply:
I noticed it was doing the wrong thing -- it's supposed to strip off the instance number, e.g. 'e1000g0'-> 'e1000g`, but instead it was stripping off at the first number (e.g. 'e' instead of 'e1000g'). It's really just for the bnx driver (without one to test, it's even unclear if the workaround is still necessary), but to make it correct, I fixed it while here.